scripts/inclusion-bot.yaml

```diff
@@ -0,0 +1,239 @@
+inclusion-bot:
+  - matches:
+      - blind to
+      - blinded to
+      - blinded by
+      - deaf to
+    alternatives:
+      - unaware of
+      - ignoring
+      - not paying attention to
+
+  - matches:
+      - crazy
+      - insane
+      - nuts
+    alternatives:
+      - hard to believe
+      - bananas
```

Comment:
not bananapants? Corey would be sad.

Reply:
I took it out based on Ashley recommendation to avoid slang, which can feel exclusionary to folks who aren't familiar with it. I think introducing people to in-language probably works better person-to-person rather than from a bot.